roomserver/internal/input/input.go

```diff
@@ -19,19 +19,19 @@ import (
 	"context"
 	"encoding/json"
 	"sync"
-	"time"
 
-	"github.com/Shopify/sarama"
+	"github.com/Arceliar/phony"
```

Comment:
I would much rather this was done in a different PR :(

Reply:
We did talk a bit about this at the time in the internal room — it pretty much boils down to the fact that buffered channels can fill up and block, unbuffered channels block anyway and `phony` was about the easiest way to ensure that we can continue to queue things up in memory without creating unexpected backpressure. The changes are limited to `input.go` (instead of managing goroutines ourselves).